ui/components/app/modals/hold-to-reveal-modal/hold-to-reveal-modal.tsx

```diff
@@ -14,9 +14,10 @@ import {
   Modal,
   ModalOverlay,
   Text,
+  ModalContent,
+  ModalHeader,
 } from '../../../component-library';
-import { ModalContent } from '../../../component-library/modal-content/deprecated';
```

Comment:
Remove two deprecated component and replace with new `ModalContent` and `ModalHeader`

Reply:
Due to styling issue, revert the code back to use `deprecated` component.